generators/src/new-test-model/new-test-model.generator.ts

```diff
@@ -188,6 +195,61 @@ export const newTestModelGenerator: CodeGenerator = {
         });
     }
 
+    // 3. Update the single package preset
```

Comment:
We need to:
1. Create a file in the source folder where we export the new model
2. Update the `package.json` to make sure we have a new entry point for the new model, we include its source in the dist package and we add the internal dependency.
3. We create the "proxy" `package.json` for the independent entry point (using `preconstruct`)
4. We install the new internal dependency